Makefile

```diff
@@ -278,6 +278,16 @@ endif
 catalog-build: opm ## Build a catalog image.
 	$(OPM) index add --container-tool docker --mode semver --tag $(CATALOG_IMG) --bundles $(BUNDLE_IMGS) $(FROM_INDEX_OPT)
 
+.PHONY: fbc-catalog-build
+fbc-catalog-build: opm ## Build a File Based Catalog (FBC) catalog image
```

Comment:
Currently nowhere.
I left it there because this will replace the catalog-build target soon-ish (see https://github.com/operator-framework/operator-sdk/issues/5832).